specs/v1.yaml

```diff
@@ -472,6 +568,57 @@ components:
           type: string
           description: The US federal employer identification number (Tax ID); unique on the system. This value should be exactly 9 digits and should not contain any special characters such as dashes.
           example: "043567500"
+        suborganizations:
+          type: array
+          items:
+            $ref: '#/components/schemas/Suborganizations'
+          description: |-
+            The list of suborganizations associated with this nonprofit.
+            Suborganizations are useful for nonprofits that have multiple chapters or locations.
+        address:
+          $ref: "#/components/schemas/Address"
+        picture:
+          type: string
+          description: The URI of the nonprofit's logo
+        website:
+          type: string
+          description: The URL of the nonprofit's website
+        createdAt:
+          type: string
+          readOnly: true
+          format: date-time
+          description: Time when this object was created. Expressed in ISO 8601 format.
+          example: "2020-07-10 15:00:00.000"
+        updatedAt:
+          type: string
+          readOnly: true
+          format: date-time
+          description: Time when this object was last updated. Expressed in ISO 8601 format.
+          example: "2021-07-11 15:34:00.000"
+    Suborganizations:
```

Comment:
the model should be singular, e.g. `Suborganization` and you should make sure to update the reference above